evm/src/executor/inspector/cheatcodes/config.rs

```diff
@@ -14,17 +14,24 @@ pub struct CheatsConfig {
     pub rpc_storage_caching: Arc<StorageCachingConfig>,
     /// All known endpoints and their aliases
     pub rpc_endpoints: Arc<RpcEndpoints>,
+
+    pub allowed_paths: Vec<PathBuf>,
```

Comment:
let's also Arc this,

as this grows, maybe it's better to Arc the struct instead all fields hmm. I assumed, that maybe we'll end up putting shared state here as well that we need to guard with `Arc<Lock<..>>` but can't think of anything.

so perhaps we use `Arc<CheatsConfig>` in the `CheatCodes` instead of arcing every field here, wdyt?